packages/mds-provider/api.ts

```diff
@@ -327,15 +320,17 @@ function api(app: express.Express): express.Express {
     return Promise.all(promises)
   }
 
-  // FIXME add pagination?
+  // TODO validate query params, 
   app.get(pathsFor('/trips'), (req: ProviderApiRequest, res: ProviderApiResponse) => {
     const { provider_id } = res.locals.claims
     log.warn(provider_id ? providerName(provider_id) : 'none', '/trips', JSON.stringify(req.params))
 
-    const { skip, take } = asPagingParams(req.query)
+    const { skip } = asPagingParams(req.query)
     const { start_time, end_time, device_id, newSkool } = req.query
 
-    // FIXME validate start_time, end_time, etc.
+    const PAGE_SIZE = 10 // set low because this is an expensive query.
+
+    const take = Number(env.PAGE_SIZE) || PAGE_SIZE
```

Comment:
I'm pretty sure forcing `take` to be a PAGE_SIZE set by the constant or an environment variable invalidates the spec -- the client should be able to specify a `take`. If we want to limit how high it can be, this we should still extract the param. Could do something like:
```typescript
const { skip, take: take_param } = asPagingParams(req.query)

const PAGE_SIZE = 10

const take = min(env.PAGE_SIZE, PAGE_SIZE, take_param)
```

Reply:
Provider stuff going to change anyway.
```
const { skip, take } = asPagingParams(req.query)
```
And then pass in:
```
take: Math.min(take, PAGE_SIZE)
```

Reply:
The client may specify the take, but all of our paging should use "sensible" defaults.